src/trading/buyer.py

```
@@ -216,3 +221,126 @@ async def _send_buy_transaction(
+            
+            tokens_received = await self._get_tokens_received_from_tx(tx_response, token_info)
+            
+            actual_price = sol_sent_to_curve / tokens_received
+            
+            logger.info(f"Bonding curve received: {sol_sent_to_curve:.6f} SOL")
+            logger.info(f"We received: {tokens_received:.6f} tokens")
+            logger.info(f"Actual execution price: {actual_price:.8f} SOL per token")
+            
+            return actual_price, tokens_received
+            
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)